apps/balancer-tools/src/app/apr/(utils)/calculatePoolStats.ts

```diff
@@ -25,54 +23,79 @@ export interface calculatePoolData extends Omit<PoolStatsData, "apr"> {
   };
 }
 
-export async function fetchPoolAveragesForDateRange(
-  poolId: string,
-  network: string,
-  from: number,
-  to: number,
-): Promise<[number, number, string, number]> {
-  // Determine if the initial date range is less than 2 days
-  const initialRangeInDays = calculateDaysBetween(from, to);
-  const extendedFrom = initialRangeInDays < 2 ? from - SECONDS_IN_DAY : from;
-
-  // Fetch snapshots within the (potentially extended) date range
-  const res = await fetchPoolSnapshots({
-    to,
-    from: extendedFrom,
-    network,
-    poolId,
-  });
-
-  let chosenData = res.poolSnapshots[0];
-
-  if (res.poolSnapshots.length == 0) {
-    const retryGQL = await fetchPoolSnapshots({
-      to: to - SECONDS_IN_DAY,
-      from: from - SECONDS_IN_DAY * 7,
+export async function fetchPoolData(poolId: string, to: number) {
```

Comment:
this fails if there is no snapshot for that day

Reply:
ok, so we need to update the snapshot table rigiht?

Reply:
now snapshot table has all day since balancer start